sconfig/shared/src/main/scala/org/ekrich/config/impl/ConfigDelayedMerge.scala

```diff
@@ -144,7 +144,7 @@ object ConfigDelayedMerge {
     last.ignoresFallbacks
   }
   // static method also used by ConfigDelayedMergeObject.
-  def render(
+  def renderAtKey(
```

Comment:
Can this be just `renderKey` so that `renderValue` for `key`, `value` makes sense?

Reply:
Sorry was just assuming it was rendering the key (name) portion and then render the value. My guess is maybe it should be `renderValue` because that it what follows key and maybe just `render` for rendering the key and the value?

From Google:
>In JSON, the "name" part of a name/value pair is called a
key. The object that follows it is the value associated with that key

Based on this you could have `renderKey` and the `render` would call `renderKey` and `renderValue`?